sdk/monitor/Azure.Monitor.Query/src/LogsQueryClientOptions.cs

```diff
@@ -52,5 +52,11 @@ internal string GetVersionString()
                 _ => throw new ArgumentException(@"Unknown version {_version}")
             };
         }
+
+        /// <summary>
+        /// Gets or sets the audience to use for authentication with Microsoft Entra ID. The audience is not considered when using a shared key.
```

Comment:
What is Entra ID?

Reply:
The new name for Azure AD. See https://learn.microsoft.com/entra/fundamentals/new-name. I'll be opening a PR soon to replace all Azure AD occurrences in the repo. It's already done for the Identity library.